File: TransCoder_tokenized_test_set_functions/python/PROGRAM_FIND_LINE_PASSING_2_POINTS.py
def lineFromPoints ( P, Q ) :
    a = Q [ 1 ] - P [ 1 ]
    b = P [ 0 ] - Q [ 0 ]
    c = a * ( P [ 0 ] ) + b * ( P [ 1 ] )
    if ( b < 0 ) :
        print ( "The line passing through points P and Q is:", a, "x ", b, "y = ", c, "\n" )
    else :
        print ( "The line passing through points P and Q is: ", a, "x + ", b, "y = ", c, "\n" )
    

File: results/java_codex/PROGRAM_FIND_LINE_PASSING_2_POINTS.java
void lineFromPoints(Pair P, Pair Q) {
  int a = Q.second - P.second;
  int b = P.first - Q.first;
  int c = a * (P.first) + b * (P.second);
  if (b < 0) {
    cout << "The line passing through points P and Q is: " << a << "x " << b << "y = " << c << endl;
  }
  else {
    cout << "The line passing through points P and Q is: " << a << "x + " << b << "y = " << c << endl;
  }
}